tests/providers/google/cloud/operators/test_gcs_to_gcs.py

```diff
@@ -28,33 +28,40 @@
 
 TASK_ID = 'test-gcs-to-gcs-operator'
 TEST_BUCKET = 'test-bucket'
-DELIMITER = '.csv'
 PREFIX = 'TEST'
+SOURCE_OBJECTS_NO_FILE = ['']
+SOURCE_OBJECTS_TWO_EMPTY_STRING = ['', '']
+SOURCE_OBJECTS_SINGLE_FILE = ['test_object/file1.txt']
+SOURCE_OBJECTS_MULTIPLE_FILES = ['test_object/file1.txt', 'test_object/file2.txt']
+SOURCE_OBJECTS_LIST = ['test_object/file1.txt', 'test_object/file2.txt', 'test_object/file3.json']
+
 SOURCE_OBJECT_WILDCARD_PREFIX = '*test_object'
 SOURCE_OBJECT_WILDCARD_SUFFIX = 'test_object*'
 SOURCE_OBJECT_WILDCARD_MIDDLE = 'test*object'
 SOURCE_OBJECT_WILDCARD_FILENAME = 'test_object*.txt'
 SOURCE_OBJECT_NO_WILDCARD = 'test_object.txt'
 SOURCE_OBJECT_MULTIPLE_WILDCARDS = 'csv/*/test_*.csv'
 DESTINATION_BUCKET = 'archive'
+DESTINATION_OBJECT = 'foo/bar'
 DESTINATION_OBJECT_PREFIX = 'foo/bar'
 SOURCE_FILES_LIST = [
     'test_object/file1.txt',
     'test_object/file2.txt',
     'test_object/file3.json',
 ]
+DELIMITER = '.json'
```

Comment:
Should be SUFFIX ?

Reply:
I actually changed my mind - I see that now we have suffix/delimiter in various places - I think it's better to keep "delimiter" from list_blob method even if it's not an obvious name.

Reply:
Ok. Well noted